File: ASPNETCore/HowTo/WebApiWPFSample/WebApiWPFSample/src/Modules/Excel/Dtos/CreateColumns.dto.cs
namespace WebApiConsoleSample
{
    public class CreateColumnsDto : BaseDto
    {
        /**The excel file name that storage manager can recognize. */
        public string ExcelPath { set; get; }
        /**The sheet name. */
        public string SheetName { set; get; }
        /**The column index. */
        public int ColumnIndex { set; get; }
        /**The count of columns. */
        public int Count { set; get; }
    }
}

File: ASPNETCore/HowTo/WebApiTypescriptSample/src/Modules/Excel/Dtos/CreateColumns.dto.ts
import { BaseDto } from "../../../Base/Base.Dto";

export class CreateColumnsDto extends BaseDto {

    /**The excel file name that storage manager can recognize. */
    public ExcelPath: string;

    /**The sheet name. */
    public SheetName: string;

    /**The column index. */
    public ColumnIndex: number;
    
    /**The count of columns. */
    public Count: number;
}